tests/torch/test_compile_torch.py

```diff
@@ -1258,6 +1259,54 @@ def test_fancy_indexing_torch(model_object, default_configuration):
     compile_brevitas_qat_model(model, x, n_bits=4, configuration=default_configuration)
 
 
+@pytest.mark.parametrize(
+    "model_object",
+    [
+        pytest.param(MultiOutputModel),
+    ],
+)
+def test_multi_output(model_object, default_configuration):
+    """Test fancy indexing torch."""
+    model = model_object()
+    x = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    y = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    torch_result = model(x[[0]], y[[0]])
+
+    # Compare with simulation
+    quantized_module = compile_torch_model(
+        model, (x, y), n_bits=4, configuration=default_configuration
+    )
+    qm_result = quantized_module.forward(x[[0]], y[[0]])
```

Comment:
would be great if the following lines could have a bit more spaces and why not comments to explain what we are checking, it's a bit hard to follow what is going on here !